### DISCUSSION TITLE
Issue In Reverifying Academic Records

### DISCUSSION THREAD

**Discussion by abdurrahman160903** [community/community#61795] [GitHub Education]
### Hi everyone,

Since some previous days I am trying to reverify my academic records to continue with the student developer package but when i try to start reverification and I select the email concerned with GitHub (already enrolled in student developer package but expired) it show a dialogue box with having a message that [abdur.................] has already been used to obtain an academic discount on GitHub account(s) ab........
Secondly, when I choose my university it shows a dialogue box and shows an error that We require applicants of CUSIT - City University of Science and Information Technology, Peshawar to use one of these school-issued email addresses to apply:

cityuniversity.edu.pk
cusit.edu.pk
Please select a different email address.
But as student of CUSIT - City University of Science and Information Technology, Peshawar we are not provided with a student email. I want to continue my student developer package with the same account but unable to reverify so kindly solve my issue as soon as possible or show my another possible solution.

**Comment by nonvenomous** [comment]
Hey there @abdurrahman160903, we and many others are in the same boat.
Unfortunately it seems like we are getting ignored.
Here a link to bigger thread which you should probably upvote and comment on to be notified for any developments, https://github.com/orgs/community/discussions/59360
